src/main/resources/modules/gallstones.json

```diff
@@ -0,0 +1,1267 @@
+              "system": "LOINC",
+              "code": "26515-7",
+              "display": "Platelet Count"
+            }
+          ],
+          "range": {
+            "low": 150,
+            "high": 450
+          }
+        }
+      ],
+      "direct_transition": "Comprehensive_Metabolic_Panel",
+      "remarks": [
+        "Expected aberrant values:",
+        "WBC (normal 4.4 - 11.3)"
+      ]
+    },
+    "RUQ_Murphy_Sign": {
+      "type": "Observation",
+      "category": "exam",
+      "unit": "",
+      "codes": [
+        {
+          "system": "LOINC",
+          "code": "80271-0",
+          "display": "Physical findings of Abdomen by Palpation"
+        }
+      ],
+      "direct_transition": "ECG",
+      "value_code": {
+        "system": "SNOMED-CT",
+        "code": 53566005,
+        "display": "Positive Murphy's Sign"
+      }
+    },
+    "Chole_Symptom4_Ends": {
+      "type": "Symptom",
+      "symptom": "Jaundice",
+      "cause": "Acute_Cholecystitis",
+      "exact": {
+        "quantity": 0
+      },
+      "conditional_transition": [
+        {
+          "transition": "Open_Recovery",
+          "condition": {
+            "condition_type": "PriorState",
+            "name": "Open_Cholecystectomy"
+          }
+        },
+        {
+          "transition": "Laparoscopic_Recovery"
+        }
+      ]
+    },
+    "Home_Recovery": {
+      "type": "Delay",
+      "range": {
+        "low": 14,
+        "high": 21,
+        "unit": "days"
+      },
+      "conditional_transition": [
+        {
+          "transition": "OC_Postoperative_Follow_up",
+          "condition": {
+            "condition_type": "PriorState",
+            "name": "Open_Cholecystectomy"
+          }
+        },
+        {
+          "transition": "LC_Postoperative_Follow-up"
+        }
+      ]
+    },
+    "Penicillin": {
+      "type": "MedicationOrder",
+      "codes": [
+        {
+          "system": "RxNorm",
+          "code": 1659149,
+          "display": "Piperacillin 4000 MG / tazobactam 500 MG Injection"
+        }
+      ],
+      "direct_transition": "Anesthesia",
+      "remarks": [
+        "Assuming the patient is not allergic to Penicillin, piperacillin is recommended for acute cholecystitis:",
+        "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2784497/"
+      ]
+    },
+    "Cefotaxime": {
+      "type": "MedicationOrder",
+      "codes": [
+        {
+          "system": "RxNorm",
+          "code": 1656318,
+          "display": "Cefotaxime 2000 MG Injection"
+        }
+      ],
+      "remarks": [
+        "For patients allergic to penicillin, a 3rd-gen cephalosporin-based antibiotic is recommended for acute cholecystitis:",
+        "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2784497/"
+      ],
+      "direct_transition": "Anesthesia"
+    },
+    "Anesthesia": {
+      "type": "CallSubmodule",
+      "submodule": "general_anesthesia",
+      "direct_transition": "Ready_for_Surgery",
+      "remarks": ""
```

Comment:
There's a bug in the module builder that causes remarks to be saved as an empty string sometimes, when it should be an array, if anything. I thought it was fixed but apparently not. There are 4 instances of `"remarks": ""` in these 2 files which will need to be deleted manually. Sorry about that.

Reply:
Done